viewing_party/party.py

```diff
@@ -1,23 +1,155 @@
 # ------------- WAVE 1 --------------------
+from ast import In
+
 
 def create_movie(title, genre, rating):
-    pass
+    #pass
+    if title and genre and rating:
+        new_movie = {"title": title , "genre": genre, "rating": rating}
+        return new_movie
+    else: 
+        return None 

```

Comment:
very clean! your solution is just fine, but if we are thinking standard practices in Python, let's consider flipping the sense of the condition around to handle the special case (something is falsy) quickly and get out, rather than having that dangling else at the end. 

This arrangement is referred to as a guard clause or a guard condition. Notice that it lets us move the main focus of the function (creating the new dict) out of an indented block, letting it stand out more clearly.

```suggestion
    if not title or not genre or not rating:
        return None
        
    new_movie = {"title": title , "genre": genre, "rating": rating}
    return new_movie

```